src/itype.h

```diff
@@ -1228,6 +1243,14 @@ struct itype {
 
         using FlagsSetType = std::set<flag_id>;
 
+    protected:
```

Comment:
I see you made a comment for this (quoted below), could you expand on what you meant? I've never had any trouble debugging in VS regardless of where a member is placed inside a struct. Do you mean scrolling through the raw data shown when you hover a variable, while the program is breakpointed?

>(P.s.: driveby change - i've moved itype::name field to be the first field in the struct to make debugging in VS less agonizing. No behavior change, no save compat issues or anything of that sort)